ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -261,6 +262,46 @@ def test_torch_matrix_power(
     )
 
 
+# vector_norm
+@handle_frontend_test(
+    fn_tree="torch.linalg.vector_norm",
+    dtype_and_x=_get_dtype_and_square_matrix(),
+    keepdim=st.booleans(),
+    ord=st.sampled_from([2.0, 1.0, 0, -1.0, -2.0, 'fro', 'nuc', None, inf, -inf]),
+    dim=st.sampled_from([None, 1, 2]),
+)
+def test_torch_vector_norm(
+    *,
+    dtype_and_x,
+    keepdim,
+    ord,
+    dim,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=dtype,
+        as_variable_flags=as_variable,
+        with_out=with_out,
+        all_aliases=["vector_norm"],
+        num_positional_args=num_positional_args,
+        native_array_flags=native_array,
+        frontend=frontend,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input=x,
+        keepdim=keepdim,
```

Comment:
`keepdim` is causing the test to throw an error since the parameter is called `keep_dim`, see more [here](https://github.com/unifyai/ivy/actions/runs/3667335176/jobs/6199733467#step:6:26853).